src/main/java/de/False/BuildersWand/events/WandEvents.java

```diff
@@ -175,7 +177,12 @@ public void playerInteract(PlayerInteractEvent event)
                 }
 
                 selectionBlock.setType(blockType);
-                selectionBlock.setData(blockSubId);
+                try {
+                    Method m = Block.class.getMethod("setData", byte.class);
```

Comment:
This method got removed later on and I was too lazy to write another NMS method for that. If you want, do that yourself lol